src/org/zaproxy/zap/extension/pscanrules/CookieHttpOnlyScanner.java

```diff
@@ -66,32 +70,56 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		if (cookies2 != null) {
 			iterator.addIterator(cookies2.iterator());
 		}
+		
+		List<String> ignoreList = new ArrayList<String>();
+		String ignoreConf = getModel().getOptionsParam().getConfig().
```

Comment:
There isnt currently an init() method (or a constructor) so I was wary of adding one just for this